assert/deno.json

```diff
@@ -25,7 +25,7 @@
     "./assert-strict-equals": "./assert_strict_equals.ts",
     "./assert-string-includes": "./assert_string_includes.ts",
     "./assert-throws": "./assert_throws.ts",
-    "./assertion-error": "./assertion_error.ts",
```

Comment:
Should `assertion-error` also be moved to `/error`? I guessed it would keep the current location

Reply:
Having an `@std/assert/assertion-error` among the other modules would look weird. I think this is fine.

Reply:
This case doesn't exactly look like a repetition of the same word to me...

Reply:
Can we discuss this case independently?

Reply:
I've reverted the change.